docs/README.md

```diff
@@ -18,6 +18,9 @@ When you are managing user experiences, it can be quite challenging to keep styl
 ## Watch the Demo on Youtube
 [![Watch the video](assets/fake_player.png)](http://youtu.be/1HREvonfqhY)
 
+## Experiment in the playground
+The folks at [\<div\>RIOTS](https://divriots.com/) created a [browser playground](https://www.style-dictionary-play.dev/) that's great if you want to experiment or test something out.
```

Comment:
Could we flip this sentence around a bit? Maybe:

```
Try the browser-based Style Dictionary playground: [https://www.style-dictionary-play.dev/](https://www.style-dictionary-play.dev/), built by the folks at [\<div\>RIOTS](https://divriots.com/)
```

Reply:
Yeah that reads better, thanks, adjusted!